helm_deploy/templates/deployment.yaml

```diff
@@ -191,6 +191,16 @@ spec:
               value: 'true'
             - name: ENABLE_PROMETHEUS_EXPORTER
               value: 'true'
+            - name: METABASE_DASHBOARD_ID
+              value: '{{ .Values.metabase.dashboardId}}'
+            - name: METABASE_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: app-secrets
+                  key: metabase_secret_key
+            - name: METABASE_SITE_URL
```

Comment:
Wondering if we should just not add the metabase siteurl into the metabase secret? Maybe this gives us more flexibility for testing?

Reply:
I'm not sure about this one - I get the idea about flexibility, but I don't think we want to start muddying up secrets and non-secrets. Ideally there'd be some UI-configurable config set-up for non-secret env vars to handle things like this.